Sanjida.html

```diff
@@ -0,0 +1,29 @@
+<html>
```

Comment:
Looks like the work i gave you last week came into this PR (Sanjida.html, blue.png, etc). We only want the work related to the task, so can you remove them?